db/schema.rb

```diff
@@ -10,10 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_09_21_233413) do
+ActiveRecord::Schema[7.0].define(version: 2022_10_11_151135) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "pg_stat_statements"
-  enable_extension "pgcrypto"
```

Comment:
Is this change expected?

Reply:
I noticed the same change locally when I dropped my DB + recreated from scratch, so I suspect there's a slight version difference in Rails or something that caused this. Maybe we make a separate PR to true-up, but as-is this seems

Reply:
No! How strange... restored in 2174aa60e